test/scala_test_testfilter/BUILD

```diff
@@ -3,4 +3,5 @@ load("@io_bazel_rules_scala//scala:scala.bzl", "scala_test")
 scala_test(
     name = "tests",
     srcs = ["A.scala", "B.scala"],
+    tags = ["manual"]
```

Comment:
Please move this target under `test_expect_failure/` instead, where it's expected to be not passing

Reply:
Done. Should the shell script remain where it is, or where do I put that?

Reply:
Oh, I think this shell script does not get executed - you need to include it into `test_rules_scala.sh`

Reply:
Should be done now.